services/like.go

```
@@ -410,6 +412,144 @@ func (ls *LikeService) StartUpdateTicker() {
+func (ls *LikeService) CancelFollow(UserID, FolloweeID string) error {
+	key := fmt.Sprintf("User:%s:followers", FolloweeID)
+	//查找关注记录以避免重复关注
+	if !utils.Client.SIsMember(utils.Ctx, key, UserID).Val() {
+		return fmt.Errorf("the user %s is not followed user %s", UserID, FolloweeID)
+	}
+	//用户移除粉丝合集
+	utils.Client.SRem(utils.Ctx, key, UserID)
+	//更新用户关注数
+	key = "User:followees"
+	utils.Client.HIncrBy(utils.Ctx, key, UserID, -1)
+	go ls.ld.DeleteFollowMessage(UserID)
```

golangci-lint (1.62.2)

Error return value of `ls.ld.DeleteFollowMessage` is not checked

(errcheck)